jpo-ode-svcs/src/main/resources/application.properties

```diff
@@ -1,61 +0,0 @@
-#General Properties
-#==================
-groupId=${project.groupId}
-artifactId=${project.artifactId}
-version=${project.version}

```

Comment:
note: these properties are now provided (or always were) by the BuildProperties bean found in OdeProperties